schema/notifications.hcl

```diff
@@ -165,3 +165,65 @@ table "notification_send_history" {
     on_delete   = CASCADE
   }
 }
+
+table "notification_silences" {
+  schema = schema.public
+  column "id" {
+    null    = false
+    type    = uuid
+    default = sql("generate_ulid()")
+  }
+  column "namespace" {
```

Comment:
Why do we need this ?

Reply:
Don't we need a namespace column for all our CRDs?